src/extensions/default/Brackets-InteractiveLinter/main.js

```diff
@@ -157,7 +153,7 @@ define(function (require, exports, module) {
 
     
 	function init(){
-		CodeInspection.register("javascript", {
+	    CodeInspection.register("javascript", {
```

Comment:
This entire function is indented incorrectly.  I want you to actually pause for a minute, and look at what's here.  There should be 4-spaces for each indent level.  Look at what you have:

```js
 	function init(){
	    CodeInspection.register("javascript", {
            name: "interactive-linter-remove-jslint",
            scanFile: $.noop
        });

        // Load up plugins and wait til they are done loading before we
        // register any handlers into Brackets
        pluginManager().done(function(plugins) {
            for (var iPlugin in plugins) {
                linterManager.registerLinter(plugins[iPlugin]);
            }

            EditorManager.on("activeEditorChange.interactive-linter", setDocument);
            setDocument(null, EditorManager.getActiveEditor());

            // If the linters change, then make sure to rebind the document with the new linter
            var lastLinters;
            preferences.on("change", function() {
                var editor = EditorManager.getActiveEditor();
                if (!editor) {
                    return;
                }

                var language = editor.document.getLanguage().getId();
                var linters  = preferences.get(language);

                if (linters && !_.isEqual(linters, lastLinters)) {
                    lastLinters = linters.slice(0);
                    handleDocumentChange();
                }
            });

        });
	}
```

Just casually looking at that, you can see that it makes no sense.  It looks like you're closing the `init()` function on the 4th line.  The actual close is way down at the bottom, but it's impossible to see, because you haven't indented the `pluginManager`'s `done` callback properly.

We've discussed this a bunch in my office, and I don't understand why we're still discussing it.  Please fix all indentation issues in this PR, push it, and then come to Github and look at it to make sure it's right and that your editor isn't lying to you.

Here's what it should look like:

```js
function init() {
    CodeInspection.register("javascript", {
        name: "interactive-linter-remove-jslint",
        scanFile: $.noop
    });

    // Load up plugins and wait til they are done loading before we
    // register any handlers into Brackets
    pluginManager().done(function(plugins) {
        for (var iPlugin in plugins) {
            linterManager.registerLinter(plugins[iPlugin]);
        }

        EditorManager.on("activeEditorChange.interactive-linter", setDocument);
        setDocument(null, EditorManager.getActiveEditor());

        // If the linters change, then make sure to rebind the document with the new linter
        var lastLinters;
        preferences.on("change", function() {
            var editor = EditorManager.getActiveEditor();
            if (!editor) {
                return;
            }

            var language = editor.document.getLanguage().getId();
            var linters = preferences.get(language);

            if (linters && !_.isEqual(linters, lastLinters)) {
                lastLinters = linters.slice(0);
                handleDocumentChange();
            }
        });
    });
}
```